Capture contributing policies in docs (#830)
ni/nimble: specs/templates/custom-component.md

### Angular integration 

*Describe the plan for Angular support, including directives for attribute binding and ControlValueAccessor for form integration. Depending on the contributor's needs, implementing Angular integration may be deferred but the initial spec should still document what work will be needed.*

### Blazor integration 

> *Describe the plan for Blazor support. See the [nimble-blazor CONTRIBUTING.md](/packages/nimble-blazor/CONTRIBUTING.md) for details. Depending on the contributor's needs, implementing Angular integration may be deferred but the initial spec should still document what work will be needed.*

Author: ```suggestion
*Describe the plan for Blazor support. See the [nimble-blazor CONTRIBUTING.md](/packages/nimble-blazor/CONTRIBUTING.md) for details. Depending on the contributor's needs, implementing Blazor integration may be deferred but the initial spec should still document what work will be needed.*
```

*Highlight any alternative implementations you considered in each section.*

> *If you think a section doesn't apply or don't know what to write, DO NOT delete it, just leave it blank and the Nimble team can help you fill it in.*

Author: ```suggestion
*If you think a section doesn't apply or don't know what to write, please DO NOT delete it. Either mark it "N/A" or just leave it blank and the Nimble team can help you fill it in.*
```

- *Keyboard Navigation and Focus*
- *Form Input*
- *Use with Assistive Technology. For example:*
  - *All components should define a role and label so that assistive technology can identify them*
  - *The implications shadow dom might have on how roles and attributes are presented in the accessibility tree.*
  - *Components which delegate focus require all global ARIA attributes to be enumerated.*

> 

Author: - *Behavior with browser configurations like "Prefers reduced motion"*

> 

Author: ```suggestion
- *Behavior with browser configurations like "Prefers reduced motion"*

```

- *Keyboard Navigation and Focus*
- *Form Input*
- *Use with Assistive Technology. For example:*

>   - *All components should define a role and label so that assistive technology can identify them*

Reviewer: ```suggestion
  - *All components should define a role and support labels / being labelled so that assistive technology can identify them*
```
